gulpfile.js

```diff
@@ -392,7 +392,6 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
             watch,
             preventRemoveAndMakeDir: opt_preventRemoveAndMakeDir,
             minify: shouldMinify,
-            wrapper: '<%= contents %>',
```

Comment:
From https://github.com/lannka/amphtml/blob/7f9aa90cf2360f84bd414542a7faf01f13c6ddfe/gulpfile.js#L1043, the default for wrapper is the same as the override you're deleting. Is this PR actually changing anything?

```
const wrapper = options.wrapper || '<%= contents %>';
```

Reply:
The code is a bit confusing to me. 

I think this is the default wrapper:
https://github.com/ampproject/amphtml/blob/master/build-system/tasks/compile.js#L117

Anyway, I tested locally, my fix worked.

Reply:
Yeah, it's a stack of calls, each of which uses a different default.

Reply:
It's probably worth mentioning this in the PR description, for posterity.

Reply:
changed the PR description.
@erwinmombay can you clarify what this code, as pointed out by @rsimha, is doing:
```
  const wrapper = options.wrapper || '<%= contents %>';

  const lazybuild = lazypipe()
      .pipe(source, srcFilename)
      .pipe(buffer)
      .pipe($$.replace, /\$internalRuntimeVersion\$/g, internalRuntimeVersion)
      .pipe($$.replace, /\$internalRuntimeToken\$/g, internalRuntimeToken)
      .pipe($$.wrap, wrapper)
      .pipe($$.sourcemaps.init.bind($$.sourcemaps), {loadMaps: true});

```

Reply:
looks like the default for dist and build diverged

https://github.com/ampproject/amphtml/blob/master/build-system/tasks/compile.js#L117

the closure default wrapper has the function wrapper, the build one doesn't.

Reply:
Shall we consolidate? The default wrapper  for `build` will leak vars to global window.

Reply:
I agree we should consolidate. If it's non-trivial, do you want to write up an issue?

Reply:
@lannka the default build probably doesn't leak since its compiled through babelify (browserify) which already adds a function wrapper. closure will hoist everything to the top level scope which is why it needs the IIFE explicitly

Reply:
probably best just to add a reason why `build` doesn't need an IIFE while `dist` does.

Reply:
@erwinmombay SG, added a comment for that